src/App.css

```diff
@@ -1,38 +1,26 @@
 .App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+  display: flex;
```

Comment:
Do we want to keep the styles in the `App` or in a separate `DayCard` file?

Reply:
In a separate `DayCard.css` file would make things clearer, I think.

Reply:
That's done!